app/services/create_account.rb

```diff
@@ -41,6 +42,10 @@ def create_redis_namespace
     CreateRedisNamespaceJob.perform_later(account)
   end
 
+  def create_default_admin_set
+    CreateDefaultAdminSetJob.perform_later(account)
```

Comment:
yeah, @mjgiarlo, intentional failure is not system-friendly.  I agree w/ @jcoyne re: failures and idempotence.  Job failure can trigger events to a separate error queue.  These should be taken seriously, not spammed.  

Our options include:
- **chaining**: defining (versions of) the jobs that know their successor, enqueuing it after their own completion.  Pro: logic.  Con: composability.
- **Tie to first user creation**: described above.  Pro: defers the timing of the problem (note: doesn't actually solve it!).  Con: conflates concerns.
- **UX step** explicitly waiting for prerequisites to complete (also above).  This also gives us space to expand if we add other components that, for whatever reason, might take longer than the couple seconds expected for a single HTTP response.